content/docs/1.0.x/install/k8s-support/index.md

```diff
@@ -45,14 +45,23 @@ Keptn versions are expressed as `x.y.z`, where `x` is the major version, `y` is
 
 * With a new Keptn release, Keptn is tested based on the default Kubernetes version of each Cloud Provider: AKS, EKS and GKE available at the release date.
 
-* Internally, a test pipeline with newer Kubernetes versions is verifying the master branch of Keptn. Known-limitations identified by these tests are referenced at the corresponding Keptn release.
+* Internally, a test pipeline with newer Kubernetes versions is verifying the master branch of Keptn. Known-limitations identified by these tests are referenced in the corresponding Keptn [Release Announcements](../../../news/release_announcements/).
 
 ## Cluster size
 
 The size of the Keptn control plane has been derived automatically and is also reported at the latest release; see *Kubernetes Resource Data* at: [https://github.com/keptn/keptn/releases](https://github.com/keptn/keptn/releases).
 
-The predefined resource values for the Keptn services are available in the [Helm Charts](https://github.com/keptn/keptn/blob/0.17.0/installer/manifests/keptn/charts/control-plane/templates/core.yaml).
+The predefined resource values for the Keptn services are available in the [Helm Charts](https://github.com/keptn/keptn/blob/{{< param "version" >}}/installer/manifests/keptn/charts/control-plane/templates/core.yaml).
```

Comment:
```suggestion
The predefined resource values for the Keptn services are available in the [Helm Charts](https://github.com/keptn/keptn/blob/{{< param "version" >}}/installer/manifests/keptn/charts/control-plane/templates/core.yaml).
```
This link gives a 404 -- it was broken when it pointed to 0.17.0 as well.  What is the correct target for "Helm Charts"?